app/src/main/java/org/simple/clinic/drugs/EditMedicinesEffectHandler.kt

```diff
@@ -34,9 +41,43 @@ class EditMedicinesEffectHandler @AssistedInject constructor(
         .addConsumer(OpenDosagePickerSheet::class.java, { uiActions.showDosageSelectionSheet(it.drugName, it.patientUuid, it.prescribedDrugUuid) }, schedulersProvider.ui())
         .addConsumer(ShowUpdateCustomPrescriptionSheet::class.java, { uiActions.showUpdateCustomPrescriptionSheet(it.prescribedDrug) }, schedulersProvider.ui())
         .addAction(GoBackToPatientSummary::class.java, uiActions::goBackToPatientSummary, schedulersProvider.ui())
+        .addTransformer(RefillMedicines::class.java, refillMedicines())
         .build()
   }
 
+  private fun refillMedicines(): ObservableTransformer<RefillMedicines, EditMedicinesEvent>? {
```

Comment:
I almost missed it, we are not supposed to return a nullable transformer here. Can you remove that, please?